pulp_python/app/utils.py

```diff
@@ -158,9 +160,9 @@ def parse_metadata(project, version, distribution):
     package["url"] = distribution.get("url") or ""
     package["sha256"] = distribution.get("digests", {}).get("sha256") or ""
     package["python_version"] = distribution.get("python_version") or package.get("python_version")
-    package["requires_python"] = distribution.get("requires_python") or package.get(
-        "requires_python"
-    )  # noqa: E501
+    package["requires_python"] = distribution.get("requires_python") or ""  # package.get(
+    #     "requires_python"
+    # )  # noqa: E501
```

Comment:
Keeping `package.get("requires_python")` returns `>=3.10` instead of `None` from `distribution.get("requires_python")`. Mixing project metadata and distribution data causes issue here.

See:
```
curl -s -H "Accept: application/vnd.pypi.simple.v1+json" https://pypi.org/simple/django/ | jq '.files[] | select(.filename == "Django-1.10.4-py2.py3-none-any.whl")'
...
"requires-python": null,
...
```
```
curl -s https://pypi.org/pypi/django/1.10.4/json | jq -r '.info.requires_python'

```
```
curl -s https://pypi.org/pypi/django/json | jq -r '.info.requires_python'
>=3.10
```
Should we always rewrite project metadata by distribution data?